<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажать для выхода</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConvert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Нажать для преобразования</string>
    </property>
    <property name="text">
     <string>Преобразовать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditData">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Ввод числа в шестнадцатиричном формате</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResultat">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>40</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Результат преобразования в двоичном формате</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Данные</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>21</y>
      <width>139</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат преобразования</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHexResultat">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>70</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
